v2/internal/pkg/imagebuilder/rebuild_catalog.go

```diff
@@ -237,7 +241,7 @@ func getStandardBuildOptions(destination string, sysCtx *types.SystemContext) (d
 		MaxPullPushRetries: 2,
 		NoCache:            true,
 		Out:                io.Discard,
-		OutputFormat:       buildah.OCIv1ImageManifest, //TODO currently red hat catalog is docker v2 format, converting it to oci is not going to cause problems?
```

Comment:
I removed this line on PR #949 so feel free to ignore it, otherwise a conflict will happen because it is the same line change.